include/zephyr/bluetooth/iso.h

```diff
@@ -621,10 +621,121 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  */
 int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf);
 
+struct bt_iso_unicast_tx_info {
+	/** The transport latency in us */
+	uint32_t latency;
+
+	/** The flush timeout in ms */
+	uint32_t flush_timeout;
+
+	/** The transport PHY  */
+	uint8_t  phy;
+
+	/** The burst number */
+	uint8_t  bn;
+
+	/** The maximum PDU size in octets */
+	uint16_t max_pdu;
```

Comment:
PHY and PDU size is already stored as part of the `struct bt_iso_chan_io_qos` struct. Should we omit them from these structs?

Reply:
I like that you get all the values from the established event in the same structure, hence I prefer this implementation. But do agree that it could be confusing that you have the value two places.